src/data/vagrant.json

```diff
@@ -74,5 +74,43 @@
 			}
 		]
 	},
-	"navigationHeaderItems": {}
+	"navigationHeaderItems": {
+		"documentation": [
+			{
```

Comment:
These icons are all the same. Not sure if all these items should be in here also, esp 'introduction'?

Reply:
Icons are good for now! 😊 And we do want to include all of the `rootDocsPaths` here. So you're good to go! ✨